package.json

```diff
@@ -13,19 +13,21 @@
     "event router"
   ],
   "scripts": {
-    "start": "yarn test",
-    "test": "jest --watchAll --no-cache --config jest.config",
     "test:ci": "jest --config jest.config --verbose",
-    "build": "npm run build:dev && npm run build:prod",
-    "build:dev": "webpack --config webpack.dev.config.js",
-    "build:prod": "webpack -p --config webpack.prod.config.js"
+    "build": "yarn build:dev && yarn build:prod",
+    "build-dev": "export NODE_ENV=dev&& webpack --display-reasons --display-error-details",
+    "build-prod": "export NODE_ENV=prod&& webpack --display-reasons --display-error-details",
+    "start": "webpack --watch",
+    "test": "jest --watchAll --no-cache --config jest.config.json"
   },
-  "main": "dist/esp.js",
+  "main": "dist/esp.ts",
```

Comment:
Comment seems to have been lost, but this is incorrect

Reply:
ah, yea that must have been some search and replace. I've put it back to `esp.js`